css/gigo.css

```diff
@@ -112,6 +112,16 @@ code {
     overflow:visible;
 }
 
+.validation-error {
+    margin-bottom: 20px;
+}
+
+.validation-error::before {
+    content: url('./images/alert.png');
```

Comment:
Done. Good to know we already have a standard icon set and I just missed it.

The rest of the review changes will have to wait a bit, as I am indeed marrying she-of-the-mad-git-skillz, this very day in fact! 💒🎉